### PR TITLE
changing instance count

### PR DESCRIPTION
**Pull request by pseudonative**

### CODE REVIEW

File: terraform/ec2-instance.tf [hunk 1]
```diff
@@ -9,7 +9,7 @@ resource "aws_instance" "myec2vm" {
     aws_security_group.vpc-ssh.id,
     aws_security_group.vpc-web.id
   ]
-  count = 3
+  count = 2
```
Comment (shoshone-dev) [inline terraform/ec2-instance.tf]: change count to 1

File: terraform/ec2-instance.tf [hunk 2]
```diff
@@ -9,7 +9,7 @@ resource "aws_instance" "myec2vm" {
...
   tags = {
     "Name" = "${var.Name}_QA_${count.index}"
```
Comment (shoshone-dev) [inline terraform/ec2-instance.tf]: change to your last name

**Review by shoshone-dev — CHANGES_REQUESTED** [review]

**Comment by shoshone-dev** [comment]
Also change the type to dev

File: terraform/ec2-instance.tf (hunk 1 above)
Comment (pseudonative) [reply terraform/ec2-instance.tf]: changed

File: terraform/ec2-instance.tf (hunk 2 above)
Comment (pseudonative) [reply terraform/ec2-instance.tf]: changed

**Review by shoshone-dev — APPROVED** [review]
/lgtm_dev

**Pull request merged**